db/seeds.rb

```diff
@@ -125,7 +122,7 @@
   # Creating Permit Applications
   puts "Seeding permit applications..."
   submitters = User.submitter
-  template_version = template1.published_template_version
+  template_version = RequirementTemplate.published.first
```

Comment:
![Screenshot 2024-03-22 at 4 27 06 PM](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/755c119f-3a1d-4697-9fad-3c8a0eb5a475)
@seeds seem to be broken.